### DISCUSSION TITLE
PCX images

### DISCUSSION THREAD

**Discussion by fwannmacher** [diasurgical/DevilutionX#3992] [Q&A] [answered]
How are the PCX images generated? I'm trying to create new images using GIMP but the images can't be loaded when I set the color mode to indexed.

**Comment by AJenbo** [comment, marked as answer]
> can't be loaded

Can you give some details here?

If you are getting issues with the colors then it's because you aren't using the games palette in your image, also make sure that you haven't set it to delete unused colors.

![image](https://user-images.githubusercontent.com/204594/151011367-472ef26d-9750-45dc-8996-1f82c0a12dff.png)

You can import the pal files directly, or copy it from one of the other pcx files.

**Reply by fwannmacher** [reply]
using the custom palette and unchecking the box solved the issue :) thanks
